src/components/NumberInput/NumberInput.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { Meta } from "@storybook/react/types-6-0";
+import { Story } from "@storybook/react";
+import NumberInput from "./NumberInput";
+import { OutlinedInputProps } from "@material-ui/core/OutlinedInput";
+
+export default {
+  title: "Components/Inputs/NumberInput",
+  component: NumberInput,
+} as Meta;
+
+const Template: Story<OutlinedInputProps> = (args) => {
+  const [val, setVal] = useState("1");
+
+  const onChange = (
+    e: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ) => {};
+
+  const onChangeUp = () => {
+    setVal((parseInt(val) + 1).toString());
+  };
+
+  const onChangeDown = () => {
+    setVal((parseInt(val) - 1).toString());
+  };
+
+  return (
+    <div style={{ width: "92px" }}>
+      <NumberInput
+        {...args}
+        value={val}
+        onChange={onChange}
```

Comment:
and this, just so anyone using it doesn't get confused.

Reply:
these comments are for the plain onchange btw, idk how to highlight on pr's or something I guess